### ISSUE TITLE
100vh not working in mobile device

### ISSUE THREAD

**Issue by tpai** [tpai/tpai.github.io#137]
You might use `100vh` very often when you are developing web app, but it will cause problems when you are using a mobile device to browse the web app.

The address bar and the navigation bar in iPhone X will change during scrolling.

![concatenate](https://user-images.githubusercontent.com/579145/80970373-9dd79a00-8e4d-11ea-92d2-dc34a793e199.png)

So the height of the center area is dynamic.

If you use `100vh`, the height will be fixed and its value is the right one, that it will cause overlay when the address bar and navigation bar show up.

In conclusion, just use `fill-available` if you don't need to take care of the Edge browser.

**References**

* https://medium.com/@littleDog/b896ac234ba9
* https://chanind.github.io/javascript/2019/09/28/avoid-100vh-on-mobile-web.html
* https://dev.to/admitkard/mobile-issue-with-100vh-height-100-100vh-3-solutions-3nae
